erpnext/crm/doctype/lead/lead_list.js

```diff
@@ -50,7 +50,16 @@ frappe.listview_settings["Lead"] = {
 		$(".list-row-container .list-row .level-right .mx-2").remove();
 		$(".list-row-container .list-row .level-right .list-row-like").remove();
 
-		$(`.result .list-row-container .list-row .level-right .list-row-activity .btn-pancake`).remove();
+  // Mask phone numbers in list view
+  const phoneCells = $('.list-row-container [data-filter^="phone,="]');
+  phoneCells.each(function() {
+      const phoneCell = $(this);
+			const phone = phoneCell.text().trim();
+			if (phone && phone.length > 5) {
```

Comment:
issue: in programming, this is called a magic number, it should be a reference constant